azext_iot/iothub/params.py

```diff
@@ -246,3 +246,88 @@ def load_iothub_arguments(self, _):
             options_list=["--content-type", "--ct"],
             help="MIME Type of file.",
         )
+
+    with self.argument_context("iot edge devices") as context:
+        context.argument(
+            "devices",
+            options_list=["--device", "-d"],
+            nargs="+",
+            action="append",
+            help="Space-separated key=value pairs corresponding to properties of the edge device to create. "
```

Comment:
what is the max amount of devices a person can add at a time?

Reply:
There is technically no defined maximum, you are only limited by input configuration / shell / how long your device arguments are, etc.